addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5695,6 +5710,16 @@ registry.SnippetMove = SnippetOptionWidget.extend(ColumnLayoutMixin, {
             });
         }
     },
+    /**
+     * @override
+     */
+    onMove() {
+        this._super.apply(this, arguments);
+        // Remove all the mobile order classes after a drag and drop.
+        [...this.$target[0].parentElement.children].forEach(el => {
+            el.className = el.className.replace(/\border(-lg)?-[0-9]+\b/g, "");
```

Comment:
Could have created a function to handle this since I see there are 3 occurrences.  But nitpicking, let's forget about it for now :+1: Can probably be handled with the task I mentioned.